pkgs/development/python-modules/scikit-misc/default.nix

```diff
@@ -0,0 +1,42 @@
+{ buildPythonPackage
+, cython
+, fetchFromGitHub
+, gfortran
+, lib
+, numpy
+, pytestCheckHook
+, python
+}:
+
+buildPythonPackage rec {
+  pname = "scikit-misc";
+  version = "0.1.4";
+
+  src = fetchFromGitHub {
+    owner = "has2k1";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1h8lxyazvq3b2c8blj131zn8wkbzjb8ynpr2a7yaanblfsbzdv92";
+  };
+
+  postPatch = ''
+    substituteInPlace pytest.ini --replace "--cov --cov-report=xml" ""
+  '';
+
+  nativeBuildInputs = [ cython gfortran numpy ];
```

Comment:
numpy is a runtime dependency https://github.com/has2k1/scikit-misc/search?q=numpy

Reply:
Just to clarify: would you prefer to see numpy removed from `nativeBuildInputs` and put in `propagatedBuildInputs` instead?